src/olympia/accounts/serializers.py

```diff
@@ -79,12 +80,13 @@ class UserProfileSerializer(PublicUserProfileSerializer):
     reviewer_name = serializers.CharField(
         min_length=2, max_length=50, allow_blank=True,
         validators=[OneOrMorePrintableCharacterAPIValidator()])
+    site_status = SiteStatusSerializer(source='*')
```

Comment:
Why do we need a `source='*'` here?

Reply:
similar to the `object()` above - when a serializer is used as a field inside a serializer it's passed some data from the parent object/data, and by default it's the name of the field.  As `UserProfile` doesn't contain a `site_status` property it fails so you have to give it a source of another field.  `source='*'` is just passing the entire UserProfile.  Could be any property.

Reply:
I see, thanks. Could you maybe add a comment above this line to explain that?

Reply:
https://github.com/mozilla/addons-server/pull/12356/commits/4acd93a499ade786efc4967b45807f2550d4a248